CONTRIBUTING.md

```diff
@@ -34,6 +34,28 @@ Failure to pass test would result in reverting a change.
 Do not hesitate to ask your colleagues if you need help or have questions.
 Post your question to Telegram or drop a line to cna-storage@vmware.com
 
+## Documentation
+
+Documentation is published to [GitHub
+Pages](https://vmware.github.io/docker-volume-vsphere/) using
+[mkdocs](http://www.mkdocs.org/).
+
+1. Documentation is updated each time a release is tagged.
+2. The latest documentation for the master can be found in
+   [docs](https://github.com/vmware/docker-volume-vsphere/tree/master/doc) in
+   markdown format
+
+To update documentation
```

Comment:
There's no real reason for a user to do this, since it's done automatically, right?

Maybe add clarification to that point or just remove step 3.